bfapi/service/jobs.py

```diff
@@ -208,7 +208,7 @@ def create(
         tide_max_24h=tide_max,
     )
 
-def __make_cli_cmd(
+def __make_algorithm_cli_cmd(
```

Comment:
In general, I prefer to make functions private unless there's some specific reason to expose them.  On code-reading, it makes it clear that that particular function is not being used elsewhere, and it cuts down on the possibility of improper external use by people who don't fully understand the context of the function.  I have had to spend too much time dealing with the code of people who only thought that their code was self-explanatory.  Context helps.